x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/utils.test.ts

```diff
@@ -24,6 +24,6 @@ describe('getBreadcrumbs', () => {
         [],
         getUrlForAppMock
       )
-    ).toEqual([{ href: 'securitySolution:detections', text: 'Detections' }]);
+    ).toEqual([]);
```

Comment:
that seems weird?

Reply:
I renamed this test scenario, and added more here  #https://github.com/elastic/kibana/pull/103275/commits/5fca23939da725984cd71a4618712f847d3bc917